src/Data/Time/PreciseDuration/Internal.purs

```diff
@@ -9,7 +9,7 @@ import Data.Decimal as Decimal
 import Data.Maybe (fromMaybe)
 
 data PreciseDuration
-  = Nanoseconds BigInt -- this is BigInt to prevent fractional nanoseconds
+  = Nanoseconds Decimal -- BigInt -- this is BigInt to prevent fractional nanoseconds
```

Comment:
Could you update this comment?

Reply:
removed

Reply:
I was thinking that we should say something about how we actually need this to be an integer and possibly how that invariant is taken care of.

Reply:
ok